src/consumer/consumer.js

```diff
@@ -7,13 +7,16 @@ import R from 'ramda';
 import chalk from 'chalk';
 import format from 'string-format';
 import partition from 'lodash.partition';
+import yn from 'yn';
 import { locateConsumer, pathHasConsumer, pathHasBitMap } from './consumer-locator';
 import { ConsumerAlreadyExists, ConsumerNotFound, NothingToImport, MissingDependencies } from './exceptions';
 import { Driver } from '../driver';
 import DriverNotFound from '../driver/exceptions/driver-not-found';
 import ConsumerBitJson from './bit-json/consumer-bit-json';
 import { BitId, BitIds } from '../bit-id';
 import Component from './component';
+import { removePrompt } from '../prompts';
```

Comment:
The consumer should never know about prompt, everything related to the prompt should be in the -cmd layer.
This is very important.